centaur/src/main/resources/standardTestCases/biscayne_read_write_functions_windows_line_endings.test

```diff
@@ -0,0 +1,26 @@
+name: biscayne_read_functions_windows_line_endings
+testFormat: workflowsuccess
+workflowType: WDL
+workflowTypeVersion: biscayne
+tags: [localdockertest, "wdl_biscayne"]
```

Comment:
I don't think we want the `localdockertest` tag here since that would prevent this from running on non-Local backends.

Reply:
Agreed. I'll remove this tag.